app/src/main/java/pm/gnosis/heimdall/data/repositories/impls/DefaultEnsRepository.kt

```diff
@@ -42,3 +45,12 @@ class DefaultEnsRepository @Inject constructor(
         private const val GET_RESOLVER = "0x0178b8bf"
     }
 }
+
```

Comment:
I don't see it here (maybe just don't understand the code), but we need to make sure that if the resolver address is zero, or if the resolved address is zero, then it is an error - we should show that address or resolver is not found, to prevent sending to the 0x000 address

Reply:
Sending any call to the 0x0 address will fail and therefore have the same result currently (that no address could be resolved).

More detailed error messages can be adjusted in a future update, but not as part of this PR